e2e/actions/ACS/site.actions.ts

```diff
@@ -0,0 +1,30 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export class SiteActions {
+
+    createSite(alfrescoJsApi, title: string, visibility: string, id?: string, description?: string) {
```

Comment:
Hi Florin I was giving a look to your PR ans honestly I guess the  e2e/actions/ACS/site.actions.ts is not necessary at the end it is only a method that wrap another method. you can directly just call the js-api from your e2e test . Those actions class are used in other test when there is some other logic around and this pattern is repeated in more then one place

Reply:
Hi Eugenio, I removed the file and updated the call accordingly. Thank you